modules/text-area/react/spec/TextArea.spec.tsx

```diff
@@ -28,7 +36,7 @@ describe('Text Area Accessibility', () => {
     const html = ReactDOMServer.renderToString(
       <>
         <label id="123">Label</label>
-        <TextArea placeholder="Placeholder" value={'Hello'} aria-labelledBy="123" />;
+        <TextArea placeholder="Placeholder" value={'Hello'} aria-labelledBy="123" onChange={cb} />;
```

Comment:
Was this just to fix a warning in tests or does this address something in this particular PR?